recipe/meta.yaml

```diff
@@ -8,37 +8,50 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 46d3fd2016cd5b842283d547877a02c418a0fe9aa7a6b0ae344115a2c990fef4
+  patches:
+    - patches/0001-rm-test-jsonpath-py.patch
```

Comment:
in place of
```
    - patches/0001-rm-test-jsonpath-py.patch
```
you could 

- `rm test/jsonpath.py` as a command before `pytest` (also for Windows).
- or do not source the whole folder in `source_files` ( a wildcard might do `test/test*.py`).
- or use a shorter patch with `--irreversible-delete` https://git-scm.com/docs/git-diff#Documentation/git-diff.txt--D (it might be harder than it looks)

Reply:
I thought we did everything through patches (where possible) as this is consistent across platforms?  `rm` is not a Windows command meaning we would need two [selected] lines, one for `win` and one `not win`.

I also have a feeling that wildcards do not work the same for Windows (certainly not in `cmd.exe` but I don't know about how Python does stuff in the recipe).

Reply:
> I thought we did everything through patches (where possible) as this is consistent across platforms?

yes, only a suggestion to shorten the patch length (but I remember that for me was painful to make `--irreversible-delete` work).

> rm is not a Windows command meaning we would need two [selected] lines, one for win and one not win.

yes you would need 2 lines, I sorted the options and put this solution as second

> I also have a feeling that wildcards do not work the same for Windows (certainly not in cmd.exe but I don't know about how Python does stuff in the recipe).

def not in `cmd.exe` but here we are in conda world. I do not remember exactly if I ever tested it. So I can't be sure.

Reply:
I believe our CI runs everything in a **Cygwin** shell, in which case `rm` should work.

Reply:
we are definitely `rm`ing files that we don't like, or simply skip them explicitly from `pytest`, also at the expense of having two selector lines.